# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: 'ViewNetworks' '2024.1.1'\n"
"Report-Msgid-Bugs-To: 'nvda-translations@groups.io'\n"
"POT-Creation-Date: 2024-06-20 15:54-0300\n"
"PO-Revision-Date: 2024-06-20 15:55-0300\n"
"Last-Translator: \n"
"Language-Team: Edilberto Fonseca <edilberto.fonseca@outlook.com>\n"
"Language: pt_BR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 3.4.4\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-Basepath: ../../../..\n"
"X-Poedit-SearchPath-0: addon/globalPlugins/ViewNetworks\n"
"X-Poedit-SearchPath-1: buildVars.py\n"

#: addon/globalPlugins/ViewNetworks/__init__.py:36
msgid "&Network Profiles"
msgstr "perfis de rede"

#: addon/globalPlugins/ViewNetworks/__init__.py:41
msgid "&Recover Password"
msgstr "&Recuperar senha"

#: addon/globalPlugins/ViewNetworks/__init__.py:46
msgid "&View Networks"
msgstr "redes &View"

#: addon/globalPlugins/ViewNetworks/__init__.py:50
msgid "Displays the networks you have connected to."
msgstr "Exibe as redes às quais você se conectou."

#: addon/globalPlugins/ViewNetworks/__init__.py:51
#: addon/globalPlugins/ViewNetworks/__init__.py:64
msgid "View Networks"
msgstr "Exibir redes"

#: addon/globalPlugins/ViewNetworks/__init__.py:55
msgid "Network Profiles"
msgstr "Perfis de rede"

#: addon/globalPlugins/ViewNetworks/__init__.py:63
msgid "Recover saved networks password."
msgstr "Recupere a senha de redes salvas."

#: addon/globalPlugins/ViewNetworks/__init__.py:68
msgid "Recover Password."
msgstr "Recuperar senha."

#: addon/globalPlugins/ViewNetworks/networks.py:31
msgid "An instance of {} is already open."
msgstr "Uma instância de {} já está aberta."

#: addon/globalPlugins/ViewNetworks/networks.py:51
msgid "Error fetching networks: "
msgstr "Erro ao buscar redes: "

#: addon/globalPlugins/ViewNetworks/networks.py:59
#: addon/globalPlugins/ViewNetworks/password.py:49
msgid "&Close"
msgstr "Fechar"

#: addon/globalPlugins/ViewNetworks/password.py:29
msgid "Enter the network name: "
msgstr "Digite o nome da rede: "

#: addon/globalPlugins/ViewNetworks/password.py:35
msgid "Key content: "
msgstr "Conteúdo da chave: "

#: addon/globalPlugins/ViewNetworks/password.py:41
msgid "&Search"
msgstr "&Pesquisar"

#: addon/globalPlugins/ViewNetworks/password.py:45
msgid "C&lean"
msgstr "Limpar"

#: addon/globalPlugins/ViewNetworks/password.py:74
#, python-brace-format
msgid "Could not find key content for network {nameWiFi}!"
msgstr "Não foi possível encontrar o conteúdo principal da rede {nameWiFi}!"

#: addon/globalPlugins/ViewNetworks/password.py:74
#: addon/globalPlugins/ViewNetworks/password.py:76
msgid "Attention"
msgstr "Atenção"

#: addon/globalPlugins/ViewNetworks/password.py:76
msgid "Error retrieving network information for {nameWiFi}: {str(e)}"
msgstr "Erro ao recuperar informações de rede para {nameWiFi}: {str(e)}"

#: addon/globalPlugins/ViewNetworks/password.py:81
msgid ""
"netsh command not found. Ensure you are running on a supported Windows "
"environment."
msgstr ""
"comando netsh não encontrado. Certifique-se de estar executando em um "
"ambiente Windows compatível."

#: addon/globalPlugins/ViewNetworks/password.py:82
#: addon/globalPlugins/ViewNetworks/password.py:86
msgid "Error"
msgstr "Erro"

#: addon/globalPlugins/ViewNetworks/password.py:86
msgid "An unexpected error occurred: {str(e)}"
msgstr "Ocorreu um erro inesperado: {str(e)}"

#: addon/globalPlugins/ViewNetworks/password.py:97
msgid "Message"
msgstr "Mensagem"

#: buildVars.py:25
msgid "List networks and recover password."
msgstr "Liste redes e recupere a senha."

#: buildVars.py:29
msgid ""
"This addon lists the networks that have been saved on this system and tries "
"when possible to recover the password."
msgstr ""
"Este addon lista as redes que foram salvas neste sistema e tenta quando "
"possível recuperar a senha."
